h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -571,6 +571,120 @@ \subsubsection{Tweedie Models}
+
+\begin{figure}[h]
+\centering
+\includegraphics[scale=0.25]{images/dispersion_param_fig1.png}
+\end{figure}
+
+\subparagraph{$\alpha (y, \phi) \text{ when } p > 2$}
+
+Here, you have,
+
+\begin{equation} \label{eq:6}
+a(y, \phi, p) = \frac{1}{\pi y}V(y,\phi, p)
+\end{equation}
+
+and $V = \sum^{\infty}_{k=1} V_k$ where
+
+\begin{equation} \label{eq:7}
+V_k = \frac{\Gamma(1+\alpha k)\phi^{k(\alpha - 1)}(p-1)^{\alpha k}}{\Gamma(1+k)(p-2)^ky^{\alpha k}}(-1)^k \sin (-k\pi \alpha)
+\end{equation}
+
+Note that $0 < \alpha < 1$ for $p > 2$. The $V_k$ terms are both positive and negative. This will limit the numerical accuracy that is obtained in summing it as shown in the following image. Again, if weights are applied to each row of the dataset, equation \ref{eq:6} becomes:
+
+\begin{equation} \label{eq:8}
+V_k = \frac{\Gamma(1+\alpha k)\phi^{k(\alpha -1)}(p-1)^{\alpha k}}{\Gamma(1+k)w^{k(\alpha -1)}(p-2)^ky^{\alpha k}}(-1)^k \sin (-k\pi \alpha)
+\end{equation}
+
+\waterExampleInR
+\lstinputlisting[style=R]{GLM_Vignette_code_examples/glm_tweedie_dispersion_example.R}
+
+\waterExampleInPython
+\lstinputlisting[style=python]{GLM_Vignette_code_examples/glm_tweedie_dispersion_example.py}
+
+\paragraph{Tweedie Variance Power Maximum Likelihood} 
+
+Depending on $p$, $y$, and $\phi$, different methods are used for this log likelihood estimation. To start, let:
+
+$$\xi = \frac{\phi}{y^{2-p}}$$
+
+If $p=2$, then it will use the log likelihood of the Gamma distribution:
+
+$$\log (p) = \begin{cases} - \infty & y=0 \\ \frac{1}{\phi} \log (\frac{1}{\phi \mu}) - \log \text{Gamma} \frac{1}{\phi} + \log (y)(\frac{1}{\phi} -1) + (-\frac{1}{\phi \mu} y) & y>0 \\\end{cases}$$
+
+If $p=3$, then it will use the inverse Gaussian distribution:
+
+$$\log (p) = \begin{cases} - \infty & y=0 \\ \frac{1}{2} \Big(-log (\phi \mu) \log (2 \pi) -3 \log \big( \frac{y}{\mu} - \frac{(\frac{y}{\mu} -1)^2}{\phi \mu \frac{y}{\mu}} \Big) - \log (\mu) & y>0 \\\end{cases}$$
+
+If $p<2$ and $\xi \leq 0.01$, then it will use the Fourier inversion method.
+
+If $p>2$ and $\xi \geq 1$, then it will also use the Fourier inversion method.
+
+Everything else will use the Series method. However, if the Series method fails (output of \texttt{NaN}), then it will try the Fourier inversion method instead.
+
+If both the Series method and Fourier inversion method fail, or if the Fourier inversion method was chosen based on the $\xi$ criterium and it failed, it will then estimate the log likelihood using the Saddlepoint approximation.
+
+Here is the general usage for Tweedie variance power maximum likelihood:
+\begin{itemize}
+\item with \texttt{fix\_tweedie\_variance\_power = False} and \\ \texttt{fix\_dispersion\_parameter = True} as it will optimize the Tweedie variance power.
+\item with \texttt{fix\_tweedie\_variance\_power = False} and \\ \texttt{fix\_dispersion\_parameter = False} as it will optimize both Tweedie variance power and the dispersion parameter.
```

Comment:
Please change

as it will optimize both Tweedie variance power and the dispersion parameter.

to

as it will estimate both the Tweedie variance power and the dispersion parameter using values set in parameter tweedie_variance_power and init_dispersion_parameter respectively.